.all-contributorsrc

```diff
@@ -2,7 +2,11 @@
   "projectName": "all-contributors",
   "projectOwner": "kentcdodds",
   "files": [
-    "README.md"
+    "README.md",
```

Comment:
@Jongjineee by adding them to the list, the bot will automatically update those files with the table too :D